flang/lib/Lower/OpenMP/ClauseProcessor.cpp

```diff
@@ -903,9 +903,12 @@ bool ClauseProcessor::processMap(
           // Explicit map captures are captured ByRef by default,
           // optimisation passes may alter this to ByCopy or other capture
           // types to optimise
+          auto location = mlir::NameLoc::get(
+              mlir::StringAttr::get(firOpBuilder.getContext(), asFortran.str()),
+              symAddr.getLoc());
           mlir::Value mapOp = createMapInfoOp(
-              firOpBuilder, clauseLocation, symAddr, mlir::Value{},
-              asFortran.str(), bounds, {},
+              firOpBuilder, location, symAddr, mlir::Value{}, asFortran.str(),
+              bounds, {},
```

Comment:
Nit: Add comments to document hardcoded arguments, same comment for the other call to `createMapInfoOp` in OpenMP.cpp.
```suggestion
              firOpBuilder, location, symAddr, /*varPtrPtr=*/mlir::Value{}, asFortran.str(),
              bounds, /*members=*/{},
```

Reply:
Hi @skatrak , Thank you for reviewing the change. This was introduced in a later commit. My merge from main , merged those changes